identifiers/country-gb/naw_consitituencies.csv

```diff
@@ -0,0 +1,41 @@
+id,name,gss_code

```

Comment:
There is a consistent typo in all filenames: "consistituencies" (sic).

Reply:
:+1:  Fixed in 048a014